backend/src/app.js

```diff
@@ -22,7 +22,7 @@ mongoose.connect(
 mongoose.Promise = global.Promise;
 
 // Setup morgan logging
-morgan.token('id', function getId(req) {
+morgan.token('id', req => {
   return req.id;
```

Comment:
You should use an implicit return here. So it should be `morgan.token('id', req => req.id);`